markdown/raspberry-pi-proximity-sensor/codelab.md

```diff
@@ -0,0 +1,471 @@
+author: Thomas Kunnumpurath
+summary: An intro to Raspberry-PI setup
+id: raspberry-pi-proximity-sensor
+tags: iot, raspberry-pi
+categories: raspberry-pi
```

Comment:
The first category determines the style on the landing page and will add a filter to the "Filter By Category" dropdown. 

I think raspberry-pi might be a bit too specific, but how about change categories to "iot,raspberry-pi" and I'll have an IoT style added to the landing page of some sort. 

For reference, right now these categories are available with styles (we've also added the list to the codelab-4-codelab): [AMQP, Boomi, Codelab, Java, JMS, Kafka, MQTT, REST, Solace, Spring]

Reply:
Done